test/entry-io.spec.js

```diff
@@ -28,16 +33,30 @@ Object.keys(testAPIs).forEach((IPFS) => {
       repo: config.defaultIpfsConfig.repo + '-entry-io' + new Date().getTime()
     })
 
+    let options, identityStore, signingStore
+
     before(async () => {
       rmrf.sync(ipfsConfig.repo)
       rmrf.sync(identityKeysPath)
       rmrf.sync(signingKeysPath)
       await fs.copy(identityKeyFixtures, identityKeysPath)
       await fs.copy(signingKeyFixtures, signingKeysPath)
-      testIdentity = await IdentityProvider.createIdentity({ id: 'userA', identityKeysPath, signingKeysPath })
-      testIdentity2 = await IdentityProvider.createIdentity({ id: 'userB', identityKeysPath, signingKeysPath })
-      testIdentity3 = await IdentityProvider.createIdentity({ id: 'userC', identityKeysPath, signingKeysPath })
-      testIdentity4 = await IdentityProvider.createIdentity({ id: 'userD', identityKeysPath, signingKeysPath })
+      const defaultOptions = { identityKeysPath, signingKeysPath }
+
+      identityStore = await storage.createStore(identityKeysPath)
+      signingStore = await storage.createStore(signingKeysPath)
+      const keystore = new Keystore(identityStore)
+      const signingKeystore = new Keystore(signingStore)
+
+      const users = ['userA', 'userB', 'userC', 'userD']
+      options = await Promise.all(users.map(async (user) => {
```

Comment:
I  believe this doesn't need to be wrapped in `Promise.all` as the function to apply (Object.assign) is sync. Can prolly remove the Promise.all wrap?